frontend/src/constants/Subvisual.js

```diff
@@ -0,0 +1,26 @@
+export const COMPANY_NAME = "Subvisual";
+export const WEBSITE = "https://subvisual.com";
+export const WHEEL_NAME = "Subvisuwheel";
+
+export const FACEBOOK = "https://facebook.com/subvisual.co";
+export const INSTAGRAM = "https://instagram.com/wearesubvisual";
+export const GITHUB = "https://github.com/subvisual";
+export const BEHANCE = "https://www.behance.net/subvisual";
+export const LINKEDIN = "https://www.linkedin.com/company/wearesubvisual";
+export const DRIBBBLE = "https://dribbble.com/subvisual";
+export const MEDIUM = "https://medium.com/subvisual";
+export const TWITTER = "https://twitter.com/subvisual";
+
+export const LOGOS = {
+  FACEBOOK: "",
+  INSTAGRAM: "",
+  GITHUB: "",
+  BEHANCE: "",
+  LINKEDIN: "",
+  DRIBBBLE: "",
+  MEDIUM: "",
+  TWITTER: "",
+};
+
+export const MAIN_COLOR = "#0000FF";
+export const SECONDARY_COLOR = "#FFFFFF";
```

Comment:
I'd probably look into doing something a bit more like the below. But this is just a quick draft, so don't just copy/paste. Think a bit about it yourself if it makes sense.


```
export const WHEEL_CONFIG = {
  name: "Subvisuwheel",
  colors: {
    main: "#0000FF",
    secondary: "#0000FF",
  },
};

export const SOCIAL = {
  behance: "https://www.behance.net/subvisual",
  dribble: "https://dribbble.com/subvisual",
  facebook: "https://facebook.com/subvisual.co",
  github: "https://github.com/subvisual",
  instagram: "https://instagram.com/wearesubvisual",
  linkedin: "https://www.linkedin.com/company/wearesubvisual",
  medium: "https://medium.com/subvisual",
  twitter: "https://twitter.com/subvisual",
};

export const SUBVISUAL = {
  channels: SOCIAL,
  website: "https://subvisual.com",
};
```

Reply:
Isn't exporting const SOCIAL and then having SOCIAL has a channel in Subvisual a bit redundant? It can be accessed in the outside as SUBVISUAL.SOCIAL.behance or SOCIAL.behance either way right?

Reply:
My intention was just to show you what's possible. And, also, I don't know what you decided about other companies? Are you including them or not?

Reply:
Because if you are, you can imagine that you'll have other wheel configs and social objects.

In that case, I'd probably only export SUBVISUAL and create local variables for `wheelConfig` and `social` 

```
const wheelConfig = {
  name: "Subvisuwheel",
  colors: {
    main: "#0000FF",
    secondary: "#0000FF",
  },
};

const social = {
  behance: "https://www.behance.net/subvisual",
  dribble: "https://dribbble.com/subvisual",
  facebook: "https://facebook.com/subvisual.co",
  github: "https://github.com/subvisual",
  instagram: "https://instagram.com/wearesubvisual",
  linkedin: "https://www.linkedin.com/company/wearesubvisual",
  medium: "https://medium.com/subvisual",
  twitter: "https://twitter.com/subvisual",
};

export const SUBVISUAL = {
  social,
  wheelConfig,
  website: "https://subvisual.com",
};
```